core/src/main/kotlin/io/specmatic/core/HttpRequestPattern.kt

```diff
@@ -30,6 +30,10 @@ data class HttpRequestPattern(
     val multiPartFormDataPattern: List<MultiPartFormDataPattern> = emptyList(),
     val securitySchemes: List<OpenAPISecurityScheme> = listOf(NoSecurityScheme())
 ) {
+    fun getHeaderKeys() = headersPattern.pattern.keys
```

Comment:
Instead of digging into headers for data, wrap this in something like `headersPattern.headerNames`

Reply:
Addressed.